gallery/src/index.html

```diff
@@ -14,7 +14,7 @@
     <link rel="stylesheet" href="gallery.css"/>
   </head>
   <body class="nx-body">
-    <div id="ui" class="nx-page"></div>
+    <div id="ui" class="nx-page--sidebar"></div>
```

Comment:
The base Block class should still be present in addition to the modifier

Reply:
> The base Block class should still be present in addition to the modifier

Done in e09f5d2899a1cf58b68b50eccda0a4319473b46b